packages/elements/src/react/sign-in/root.tsx

```diff
@@ -94,6 +94,7 @@ export function SignInRoot({
 
   // TODO: eventually we'll rely on the framework SDK to specify its host router, but for now we'll default to Next.js
   const router = useNextRouter();
+  const isRootPath = path === router.pathname();
```

Comment:
This is required, on both Sign Up and Sign In, otherwise, in the case of callbacks, the fallback will show regardless.